build.gradle

```diff
@@ -118,6 +118,5 @@ publishing {
 }
 
 wrapper {
-	gradleVersion = '5.0-rc-1'
```

Comment:
Why did you remove the version rather than just update it to the version you want?

Reply:
Because some version of the wrapper plugin introduced the possibility to provide the gradle version from the command line (`--gradle-version`) and it's the way they recommend to upgrade the version now: `./gradlew wrapper --gradle-version=xxx`.

I was trying to avoid compatibility problems, but if it still works... why not.

Doc of Gradle 2.0, the version is in `build.gradle`: https://docs.gradle.org/2.0/userguide/gradle_wrapper.html
Doc of Gradle 5.6.3, the version is provided on the command line: https://docs.gradle.org/5.6/userguide/gradle_wrapper.html#sec:adding_wrapper

It seems to make sense, too, since the version is only used to *upgrade* the plugin, and once it's upgraded you can safely remove it because it's duplicated somewhere in a properties file.